docs/running-calyx/profiler.md

```diff
@@ -0,0 +1,37 @@
+# Calyx Profiler
+
+*Note: The profiler is still in development. If you have any suggestions, thoughts, or feedback, please let us know!*
+
+Profilers can help you analyze performance information to find places you can optimize in your code. Currently, the Calyx profiler prouduces timing information in terms of cycle counts.
+
+## Setup
+
+In order to run the profiler, you need:
+- [fud2][]
+- The Python `vcdvcd` library
```

Comment:
how are people supposed to get this library? could `fud2` automatically install it in the virtual environment it creates?

Reply:
I linked to the repo and suggested an installation command, how does the new bullet look?

Reply:
@EclecticGriffin do you know if there is a way to have this python library installed when running `fud2 env init`? I think that would be the most straightforward way for people to get this dependency.

Reply:
Ah I didn't know about `fud2 env init`, I'll add `vcdvcd` to the init commands!

Reply:
Added the install command here: https://github.com/calyxir/calyx/pull/2370/files#diff-8f6eee891971328771ea1488c38a76b5722005c2e857008c8289fc223f0040b9

Reply:
That is great, thanks!